### PR TITLE
feat: deleteProjectImageAsync

### PR DESCRIPTION
**Pull request by sstangeland**

### CODE REVIEW

File: src/http/apiClients/OrgClient.ts [hunk 1]
```diff
@@ -90,6 +90,21 @@ export default class OrgClient extends BaseApiClient {
         );
     }
 
+    public async deleteProjectImageAsync(projectId: string): Promise<HttpResponse<void>> {
+        const orgResources = this.resourceCollections.org;
+        const url = orgResources.project(projectId + '/image');
```
Comment (eskil0312) [inline src/http/apiClients/OrgClient.ts:95]: Add a new method for image in resource collection, use the combineUrls helper function

Comment (sstangeland) [reply src/http/apiClients/OrgClient.ts]: Added projectImage method in OrgResourceCollection in commit 9bf02d1

File: src/http/resourceCollections/OrgResourceCollection.ts [hunk 2]
```diff
@@ -134,6 +134,10 @@ export default class OrgResourceCollection extends BaseResourceCollection {
         return combineUrls(this.getContractPositionsUrl(projectId, contractId), positionId);
     }
 
+    projectImage(projectId: string) {
+        return combineUrls(this.getBaseUrl(), 'projects', projectId, 'image');
```
Comment (eskil0312) [inline src/http/resourceCollections/OrgResourceCollection.ts:138]: In here you can use the project method? combineUrls(this.project(projectId), 'image')?

Comment (sstangeland) [reply src/http/resourceCollections/OrgResourceCollection.ts]: Yes, that makes sense, using it now

**Review by eskil0312 — APPROVED** [review]

**Pull request merged**